SharpPcap/LibPcap/LibPcapLiveDevice.cs

```diff
@@ -164,6 +164,24 @@ public override void Open(DeviceConfiguration configuration)
                 LibPcapSafeNativeMethods.pcap_set_promisc(PcapHandle, (int)(mode & DeviceModes.Promiscuous));
                 LibPcapSafeNativeMethods.pcap_set_timeout(PcapHandle, configuration.ReadTimeout);
 
+                if(configuration.TimestampResolution != null)
+                {
+                    int retval = LibPcapSafeNativeMethods.pcap_set_tstamp_precision(PcapHandle, (int)configuration.TimestampResolution);
+                    if(retval != 0)
+                    {
+                        throw new InvalidOperationException("pcap_set_tstamp_precision() failed - return value " + retval);
```

Comment:
You may want to change this once #220 is merged with new configuration failure handling